<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteBookPage</class>
 <widget class="QDialog" name="deleteBookPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="titleEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
